app/_landing_pages/konnect-reference-platform.yaml

```
@@ -0,0 +1,114 @@
+metadata:
+  title: "{{site.konnect_short_name}} Reference Platform"
+  content_type: landing_page
+  products:
+    - api-ops
+  description: |
+    Provides a complete guide for platform builders to to integrate {{site.konnect_product_name}} into their engineering 
+    organization and API delivery process. 
+  tags:
+    - konnect
+    - automation
+    - apiops
+  breadcrumb: /konnect-reference-platform/
+
+rows:
+  - header:
+      type: h1
+      text: "{{site.konnect_short_name}} Reference Platform"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    The {{site.konnect_short_name}} Reference Platform is a technical guide designed to help you integrate 
+                    [{{site.konnect_product_name}}](https://konghq.com/products/kong-konnect)
+                    into your engineering organization's technology stack. The platform provides an opinionated framework that will aid you in 
+                    designing your {{site.konnect_short_name}} organization and the building of a _Federated APIOps_ delivery process with 
+                    governance, auditing, and GitOps based automations. The platform provides tools, examples, best practices, and documentation 
+                    to guide you in on-boarding with {{site.konnect_short_name}}. You can choose how to use the components, 
+                    individually or collectively, to fit your needs and engineering requirements.
```

```suggestion
                    governance, auditing, and GitOps based automations. 
                    
                    The platform provides resources to guide you in onboarding with {{site.konnect_short_name}}:
                    * Tools
                    * Examples
                    * Best practices
                    * Extensive documentation 
                   You can choose how to use the components, 
                    individually or collectively, to fit your needs and engineering requirements.
```
Trying to break up the wall of text a bit.

Maybe even something like "Custom-built" or "purpose-built tools"; "real life examples" to help flesh it out a bit.